backend/api/src/main/kotlin/io/tolgee/security/JwtTokenFilter.kt

```diff
@@ -48,6 +50,9 @@ class JwtTokenFilter @Autowired constructor(
             throw PermissionException(Message.EXPIRED_SUPER_JWT_TOKEN)
           }
         }
+        if (this.needsToBeServerAdmin(req)) {
```

Comment:
There should be special test for this. When the annotation stops working it's huge security issue. I don't also think that this is right place to do this. Because being server admin is not directly related to JWT token. You can be server admin while logging in with PAK or PAT and then this whole check might be bypassed.

I actually guess that currently you can call annotated endpoints method with **ANY** PAT token and will be executed